[2024-07-03T09:57:08.471+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T09:57:08.482+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T09:57:08.482+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:08.510+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:50:00+00:00
[2024-07-03T09:57:08.515+0000] {standard_task_runner.py:57} INFO - Started process 20465 to run task
[2024-07-03T09:57:08.522+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:50:00+00:00', '--job-id', '314', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpwenelxpz']
[2024-07-03T09:57:08.531+0000] {standard_task_runner.py:85} INFO - Job 314: Subtask id_task_get_emails
[2024-07-03T09:57:08.607+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:08.703+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:50:00+00:00'
[2024-07-03T09:57:08.705+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:08.705+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:08.717+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T055000, start_date=20240703T095708, end_date=20240703T095708
[2024-07-03T09:57:08.730+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 314 for task id_task_get_emails (name 'json' is not defined; 20465)
[2024-07-03T09:57:08.779+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:08.800+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:04:45.549+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T12:04:45.567+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T12:04:45.567+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:04:45.591+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:50:00+00:00
[2024-07-03T12:04:45.596+0000] {standard_task_runner.py:57} INFO - Started process 23285 to run task
[2024-07-03T12:04:45.603+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:50:00+00:00', '--job-id', '756', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpjfk_jjz1']
[2024-07-03T12:04:45.605+0000] {standard_task_runner.py:85} INFO - Job 756: Subtask id_task_get_emails
[2024-07-03T12:04:45.696+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:04:45.888+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:50:00+00:00'
[2024-07-03T12:04:45.889+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:04:45.890+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:45.890102_User@StatDep: EmailRobot in screated.
[2024-07-03T12:04:45.890+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:45.890483_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:04:45.890+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:45.890804_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:04:46.318+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:04:46.717+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:04:47.126+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:04:47.126573_conneciton to db is set.
[2024-07-03T12:04:47.137+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:04:47.165+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T055000, start_date=20240703T120445, end_date=20240703T120447
[2024-07-03T12:04:47.211+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 756 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23285)
[2024-07-03T12:04:47.241+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:04:47.283+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:22:19.726+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T12:22:19.757+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [queued]>
[2024-07-03T12:22:19.757+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:22:19.795+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:50:00+00:00
[2024-07-03T12:22:19.815+0000] {standard_task_runner.py:57} INFO - Started process 24104 to run task
[2024-07-03T12:22:19.824+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:50:00+00:00', '--job-id', '1032', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpei7lbr79']
[2024-07-03T12:22:19.826+0000] {standard_task_runner.py:85} INFO - Job 1032: Subtask id_task_get_emails
[2024-07-03T12:22:19.911+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:22:20.031+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:50:00+00:00'
[2024-07-03T12:22:20.032+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:22:20.032+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:20.032801_User@StatDep: EmailRobot in screated.
[2024-07-03T12:22:20.033+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:20.033002_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:22:20.033+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:20.033140_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:22:20.406+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:22:20.667+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:22:20.667+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:20.667769_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:22:20.668+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:22:20.724+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T055000, start_date=20240703T122219, end_date=20240703T122220
[2024-07-03T12:22:20.785+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:22:20.827+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
